plugins/kubernetes-backend/alpha-api-report.md

```diff
@@ -5,7 +5,7 @@
 ```ts
 import { BackendFeature } from '@backstage/backend-plugin-api';
 
-// @alpha
+// @public
 const kubernetesPlugin: () => BackendFeature;
```

Comment:
Think that you will want to move this from the `/alpha` exports if you're wanting to promote it to public API, otherwise this should stay as `@alpha` annotation.

Reply:
Thanks Ben for your feedback. Ee will keep it as alpha for now, I'll made the changes  cc @jamieklassen